dist/ramda.js

```diff
@@ -1,3 +1,8 @@
+//  Ramda v0.9.1
+//  https://github.com/ramda/ramda
+//  (c) 2013-2015 Scott Sauyet and Michael Hurley
+//  Ramda may be freely distributed under the MIT license.
+
```

Comment:
No worries. Order will be restored if and when this pull request is merged. :)

I hope you're well again soon.